extensions/liquibase/deployment/src/main/java/io/quarkus/liquibase/LiquibaseProcessor.java

```diff
@@ -96,6 +97,7 @@ void nativeImageConfiguration(
                 liquibase.sql.visitor.SqlVisitor.class);
 
         // load the liquibase services
+        Map<String, List<String>> serviceClassesImplementationRegistry = new HashMap<>();
```

Comment:
Should we care about the order here (`LinkedHashMap`) ?

Reply:
> Should we care about the order here (`LinkedHashMap`) ?

Probably yes, to avoid uncertainty while loading each service, WDYT?

Reply:
> > Should we care about the order here (`LinkedHashMap`) ?
> 
> Probably yes, to avoid uncertainty while loading all services, WDYT?

I do not think so. I think we just need a lookup table here, order is irrelevant. What uncertainty are you thinking of?

Reply:
Ah nevermind, I thought you were iterating entries. I should stop reviewing code late at night :stuck_out_tongue: